prow/cmd/checkconfig/main_test.go

```diff
@@ -1019,3 +1020,85 @@ func TestVerifyOwnersPresence(t *testing.T) {
 		})
 	}
 }
+
+func TestOptions(t *testing.T) {
+
+	var defaultGitHubOptions flagutil.GitHubOptions
+	defaultGitHubOptions.AddFlagsWithoutDefaultGitHubTokenPath(flag.NewFlagSet("", flag.ContinueOnError))
+
+	StringsFlag := func(vals []string) flagutil.Strings {
+		var flag flagutil.Strings
+		for _, val := range vals {
+			flag.Set(val)
+		}
+		return flag
+	}
+
+	testCases := []struct {
+		name     string
+		args     []string
+		expected *options
```

Comment:
We should return an error on failure and assert that we get one in testing

Reply:
Can you elaborate?

Reply:
I mean, let's be explicit that we are expecting an error (regardless of what output object we expect) and assert that we get one in that case